sklearn/datasets/base.py

```diff
@@ -255,14 +255,25 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
+    if return_X_y == false
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    if return_X_y == true
```

Comment:
After making the changes, I generated the docs by running ```make html-noplot``` in```doc/``` folder.
Path of html page: ```file:///home/manu/github/scikit-learn/doc/_build/html/stable/modules/generated/sklearn.datasets.load_iris.html#sklearn.datasets.load_iris```

What is it, I might be doing wrong ?